### DISCUSSION TITLE
Is it possible to set `CHATGPT_BASE_URL` to the official api?

### DISCUSSION THREAD

**Discussion by rakuyoMo** [acheong08/ChatGPT#1195] [Q&A]
If I deploy V1 on an overseas, not GFW-restricted server, can I set `CHATGPT_BASE_URL` directly to `https://chat.openai.com/backend-api/`?

If so, this should prevent the chatgpt account from being blocked

**Comment by acheong08** [comment]
Requires cloudflare bypass. I don't know how to implement TLS spoofing + streaming within pure python.

**Reply by rakuyoMo** [reply]
try [cloudscraper](https://github.com/VeNoMouS/cloudscraper) ?

**Reply by acheong08** [reply]
Doesn't work. Tried
